### ISSUE TITLE
Lack of a license

### ISSUE THREAD

**Issue by orowith2os** [tinywrkb/flatpaks#5]
Probably best to add a license (like the CC0) to have some clear terms on which people can utilize the manifests published here. Without one, any content is considered proprietary by platforms such as GitLab.

**Comment by tinywrkb** [comment]
I don't believe that a packaging recipe is copyright protected, and it would be impossible to enforce its licensing and detect possible violators.  
In other words, a license here would be meaningless.

The packaging was initially heavily influenced by Arch Linux official and AUR packages. These don't have or require a license, so I see no reason to add one.

> Without one, any content is considered proprietary by platforms such as GitLab.
In my opinion, that's a broken design, a policy shouldn't be embedded in the platform. I'm not going to add a license because of this.

The actual code here comes as patch files, and those have git commit message and author details, or a link to where it came from, so the original license should be applied.

Icons mainly come from the upstream source, or official distributed binaries, so having them here for packaging falls under fair use.  
Icons that were generated from official media (e.g. website, social media) have a link to the source in the packaging manifest, and this also falls under fair use.  
Icons that come from a third party, e.g. Wikipedia page, also have a link to the source, and those that were chosen have a suitable license for packaging and distributing.

**Comment by orowith2os** [comment]
The issue is less enforcing the licensing, more making sure no potential issues arise. I would appreciate some clear terms on which I can use the recipes.

**Comment by tinywrkb** [comment]
You can treat the packaging as public domain, and use it as you like.  
In the following weeks, I will go through everything here as part of bumping the runtime to 22.08, and I plan to look at adding attributions where packaging was more involved than just some configure options, and followed a distro packaging.  
After this, I will feel more comfortable to add a license.  
Attribution to distro packaging is not really required, but it is more courteous.
